src/pydata_sphinx_theme/assets/styles/sections/_header.scss

```diff
@@ -159,17 +187,11 @@
 
   // Toggle buttons
   button.sidebar-toggle {
-    display: flex;
-    cursor: pointer;
     font-size: var(--pst-font-size-icon);
-    align-items: center;
     color: var(--pst-color-muted);
     margin-bottom: 0;
-    padding-bottom: 0.25rem;
     background-color: inherit;
-    border: none;
-
-    @include icon-navbar-hover;
+    padding: 0.5rem;
```

Comment:
this is a substantive change (not just refactoring). Just flagging for other reviewers; I checked the [PR build](https://pydata-sphinx-theme--1866.org.readthedocs.build/en/1866/api/pydata_sphinx_theme/index.html) versus [current main](https://pydata-sphinx-theme.readthedocs.io/en/latest/api/pydata_sphinx_theme/index.html) and I'm happy with the change (need to view in narrow viewport / on mobile to see the difference)